wa/workloads/speedometer/__init__.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, target, **kwargs):
 
     def setup(self, context):
         super(Speedometer, self).setup(context)
-        self.target.execute('am start -a android.intent.action.VIEW -d http://browserbench.org/Speedometer/')
+        self.target.execute('am start -a android.intent.action.VIEW -d http://browserbench.org/Speedometer2.0/')
```

Comment:
If there are going to be multiple versions of the benchmark might it be worth adding a parameter so that the version can be selected?